projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_router.py

```diff
@@ -59,6 +64,17 @@ def send_object_for_ingestion(
         target: Optional[str] = None,
         collection_id: Optional[str] = None,
     ):
+        # Initialize the pre- and post- processors.
+        if not self._initialized_pre_processors:
+            self._initialized_pre_processors = self.__get_initialized_processors(
+                "INGEST_PAYLOAD_PREPROCESS_SEQUENCE"
+            )
+
+        if not self._initialized_post_processors:
+            self._initialized_post_processors = self.__get_initialized_processors(
```

Comment:
I see one more initialisation here, on each `send_tabular_data_for_ingestion()`,

yet both send_*_for_ingestion() methods use cached infesters if available, therefore initialising the ingesterbase once - yet processors are reinitialised each time it seems to me?

Reply:
Fixed, thanks